libs/server/Resp/Bitmap/BitmapCommands.cs

```diff
@@ -432,10 +431,73 @@ private bool StringBitPosition<TGarnetApi>(byte* ptr, int count, ref TGarnetApi
         /// <summary>
         /// Performs bitwise operations on multiple strings and store the result.
         /// </summary>
-        private bool StringBitOperation<TGarnetApi>(int count, byte* ptr, BitmapOperation bitop, ref TGarnetApi storageApi)
+        private bool StringBitOperation<TGarnetApi>(int count, byte* ptr, ref TGarnetApi storageApi)
             where TGarnetApi : IGarnetApi
         {
+            // <AND | OR | XOR | NOT> destkey key
+            if (count < 3)
```

Comment:
I think this is good work, but arity checks are not part of this PR. Even though we need to perform these checks earlier, maybe by capturing a parsing state.

Reply:
Agreed in general, but parsing state should probably be out of scope for this PR. Adding checks where we notice them missing is fine for now.